libcontainer/rootfs_linux.go

```diff
@@ -144,11 +147,30 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		tmpFilePath := ""
 		stat, err := os.Stat(dest)
 		if err != nil {
 			if err := os.MkdirAll(dest, 0755); err != nil {
 				return err
 			}
+		} else {
+			if copyUp {
+				tarStream, err = archive.Tar(dest, archive.Uncompressed)
+				if err != nil {
+					return fmt.Errorf("failed to tar archive: %v", err)
+				}
+				tmpFile, err := ioutil.TempFile("/run", "runctmpfile")
```

Comment:
Yes, it is.

Sent from my iPhone

> On Mar 30, 2016, at 5:45 PM, Michael Crosby <notifications@github.com> wrote:
> 
> In libcontainer/rootfs_linux.go:
> 
> >  		stat, err := os.Stat(dest)
> >  		if err != nil {
> >  			if err := os.MkdirAll(dest, 0755); err != nil {
> >  				return err
> >  			}
> > +		} else {
> > +			if copyUp {
> > +				tarStream, err = archive.Tar(dest, archive.Uncompressed)
> > +				if err != nil {
> > +					return fmt.Errorf("failed to tar archive: %v", err)
> > +				}
> > +				tmpFile, err := ioutil.TempFile("/run", "runctmpfile")
> This is using /run on the host right?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>